DancingGoat/Content/Styles/EditMode.less

```diff
@@ -69,7 +66,11 @@
             }
         }
     }
-} 
+}
+
+.edin-in-cloud-button {
```

Comment:
edin

Reply:
also what exactly does edit in cloud mean?

Reply:
I'm not sure I understand - it's button navigating to listing of current project.
```
> <div class="edin-in-cloud-button">
>   <a target="_blank" href="https://app.kenticocloud.com/ddd5e846-5685-0067-5338-c56954c0bd7c" class="btn-cloud btn-cloud__padding btn-cloud-info">
>     <i class="btn-pen"></i> Edit in Kentico Cloud
>   </a>
> </div>
```

Reply:
Fixed